pkg/system/phase2_creating.go

```diff
@@ -168,13 +171,49 @@ func (r *Reconciler) SetDesiredCoreApp() {
 	}
 }
 
-// ReconcileCredentialsRequest creates a CredentialsRequest resource if neccesary and returns
+// ReconcileBackinStoreCredentials creates a CredentialsRequest resource if neccesary and returns
 // the bucket name allowed for the credentials. nil is returned if cloud credentials are not supported
-func (r *Reconciler) ReconcileCredentialsRequest() error {
-	if !util.IsAWSPlatform() {
-		r.Logger.Info("not running in AWS. skipping ReconcileCredentialsRequest")
-		return nil
+func (r *Reconciler) ReconcileBackinStoreCredentials() error {
+	if util.IsAWSPlatform() {
+		return r.ReconcileAWSCredentials()
+	}
+	return r.ReconcileCephObjectstoreCredentials()
+
+}
+
+// ReconcileCephObjectstoreCredentials creates a ceph objectstore user if a ceph objectstore exists in the same namespace
+func (r *Reconciler) ReconcileCephObjectstoreCredentials() error {
```

Comment:
```suggestion
func (r *Reconciler) ReconcileRGWCredentials() error {
```

Reply:
It is a pretty long name😩 I'll change it